### DISCUSSION TITLE
I rewrote my WakeOnLan project using PocketBase

### DISCUSSION THREAD

**Discussion by seriousm4x** [pocketbase/pocketbase#1715] [Show and tell]
TLDR: I rewrote my WakeOnLan project [UpSnap](https://github.com/seriousm4x/UpSnap) from Python to Go using PocketBase.

I started programming in Python about 3 years ago and have been using Go more and more the last few months. I discovered PocketBase and found it perfect for UpSnap, a web app to wake up/shutdown devices via wake on lan and cron. I had used Svelte quite a bit before anyway, so everything worked very well. Here and there my code is still a bit spaghetti, but I made an effort ;) Feel free to check [out the repo](https://github.com/seriousm4x/UpSnap) and many thanks to the PocketBase team!
